integrations/axum/Cargo.toml

```diff
@@ -11,28 +11,28 @@ edition.workspace = true
 [dependencies]
 any_spawner = { workspace = true, features = ["tokio"] }
 hydration_context = { workspace = true }
-axum = { version = "0.8.1", default-features = false, features = [
+axum = { default-features = false, features = [
```

Comment:
This introduces a very subtle bug. The workspace dependency is enabled with default features, but here we import it without default features. Since cargo features are only additive, this might enable the default feature for this dependency. Almost all dependencies in the workspace level should be set to `default-features = false` and the default features should be enabled on a per-member level.

You might want to confirm this isn't the case here

Reply:
Thanks for mentioning it! The only problematic part is `default-features` as it is already turned on by default, and members cannot turn it off. We can simply inspect it with : `cargo metadata --format-version=1 | jq '.packages[] | select(.name == "leptos_axum") | .dependencies[]'`.